packages/DrillDownTable/src/components/Pagination/index.tsx

```diff
@@ -78,7 +94,10 @@ RevealPagination.defaultProps = defaultPaginationProps;
 
 export { RevealPagination };
```

Comment:
Does the naming mean that it can only be used with the Reveal project?

Reply:
@p-netm ^^

Reply:
yeah renaming, thanks.